specification/msi/resource-manager/Microsoft.ManagedIdentity/stable/2018-11-30/ManagedIdentity.json

```diff
@@ -254,7 +254,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/IdentityPatch"
+              "$ref": "#/definitions/IdentityUpdate"
```

Comment:
This is a breaking change on an existing stable API-version and should be reverted. [See error details](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=257293&view=logs&j=b6885ff5-5b63-57e7-07bc-9d8d90d734f1&t=982b7c5c-6d4b-53e9-42d9-af8a8061e6e5&l=3093) 

You should skip modifying stable version of API.

Reply:
This was just added in this: https://github.com/Azure/azure-rest-api-specs/pull/7528

I saw that other specs had operationXUpdate and wanted to stay consistent with most of Azure.

Reply:
As you can see that .NET SDK automation failed due to breaking changes by this. This kind of change should really be avoided. And I apologize for missing the previous rename in the 7528. Can you please change it back to **Identity**?

Reply:
Discussed offline. Needs to split the model because of PATCH support.